sql/core/src/test/scala/org/apache/spark/sql/execution/vectorized/ColumnarBatchSuite.scala

```diff
@@ -1311,4 +1314,172 @@ class ColumnarBatchSuite extends SparkFunSuite {
     batch.close()
     allocator.close()
   }
+
+  test("CachedBatch boolean Apis") {
```

Comment:
move these to a new test suite

Reply:
I see. Moved them into `ColumnVectorSuite`